README.md

```diff
@@ -29,25 +48,30 @@ addEventListener('fetch', event => {
   event.respondWith(handleEvent(event))
 })
 
-const customKeyModifier = url => {
-  //custom key mapping optional
-  if (url.endsWith('/')) url += 'index.html'
-  return url.replace('/docs', '').replace(/^\/+/, '')
-}
-
 async function handleEvent(event) {
   if (event.request.url.includes('/docs')) {
     try {
-      return await getAssetFromKV(event, { mapRequestToAsset: customKeyModifier })
+      return await getAssetFromKV(event)
     } catch (e) {
-      return new Response(`"${customKeyModifier(event.request.url)}" not found`, {
-        status: 404,
-        statusText: 'not found',
-      })
+      if (resp instanceof KVError) {
+        switch (e.status) {
```

Comment:
Ooh! I thought we were going to match over error types (and I honestly did prefer that approach!) I feel like matching on types like NotFoundError would look nice in the docs, too, especially because we describe the error types right below this example.